_CI/scripts/graph.py

```diff
@@ -47,7 +47,7 @@ def graph():
                             '-o png '
                             '-A '
                             '-f PUB_ONLY '
-                            '-p graphs {}').format(os.path.join('..', f'{PROJECT_SLUG}'))
+                            f'-p graphs {os.path.join("..", f"{PROJECT_SLUG}")}')
```

Comment:
Since we are doing the improvement, maybe go all the way and use Pathlib here instead?